docs/driver-parameters.md

```diff
@@ -8,6 +8,7 @@ Name | Meaning | Example | Mandatory | Default value
 --- | --- | --- | --- | ---
 skuName | Azure file storage account type (alias: `storageAccountType`) | `Standard_LRS`, `Standard_ZRS`, `Standard_GRS`, `Standard_RAGRS`, `Premium_LRS` | No | `Standard_LRS` <br><br> Note:  <br> 1. minimum file share size of Premium account type is `100GB`<br> 2.[`ZRS` account type](https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy#zone-redundant-storage) is supported in limited regions <br> 3. Premium files shares is currently only available for LRS
 storageAccount | specify Azure storage account name| STORAGE_ACCOUNT_NAME | - No for SMB share </br> - Yes for NFS share|  - For SMB share: if empty, driver will find a suitable storage account that matches `skuName` in the same resource group; if a storage account name is provided, storage account must exist. </br>  - For NFS share, storage account name must be provided
+EnableLargeFileShares | Specify whether to use a storage account with large file shares enabled or not. If this flag is set to true and a storage account with large file shares enabled doesn't exist, a new storage account with large file shares enabled will be created  | `true`,`false` | No | `true`
```

Comment:
Let's be consistent with the rest of the driver parameters and being with a lowercase letter.